endpoint.go

```diff
@@ -723,14 +723,6 @@ func (ep *endpoint) sbLeave(sb *sandbox, force bool, options ...EndpointOption)
 }
 
 func (n *network) validateForceDelete(locator string) error {
-	if n.Scope() == datastore.LocalScope {
-		return nil
-	}
```

Comment:
yeah.. I misread it (thought we want to block force disconnect for local scope networks).